src/codegen/translate_from_glib.rs

```diff
@@ -81,22 +81,27 @@ impl TranslateFromGlib for analysis::return_value::Info {
 }
 
 fn from_glib_xxx(transfer: library::Transfer, array_length: Option<&String>) -> (String, String) {
+    let good_print = |name: &str| {
+        if name == "error" {
+            format!(", {} as usize", name)
+        } else {
+            format!(", {{ let {0} = {0}.assume_init(); {0} as usize }})", name)
```

Comment:
Seems this can be just `format!(", {}.assume_init() as usize)", name)` without "error" branch

Reply:
@GuillaumeGomez If you don't want this one-liner, please, merge this PR yourself.

Reply:
Oh, error branch still needed.
Can you tell me which function?

Reply:
I updated it to a one-liner. However, I don't understand your last comment. What error?

Reply:
I meant branch `if name == "error"`, I suspect that it never reached:
note that it not ends with ')', 
and I updated it to produce bad code, but gtk-rs crates still builds fine after regen.

Reply:
I'll remove this check and see the result.